tendermint/src/lib.rs

```diff
@@ -26,6 +26,7 @@ extern crate prost_amino as prost;
 #[allow(unused_extern_crates)]
 #[macro_use]
 extern crate prost_amino_derive as prost_derive;
+extern crate bytes;
```

Comment:
Are either of these `extern crate`s still needed? I know `prost` was looking directly in the `sysroot` for them before which precluded renaming them, but if everything is namespaced correctly, it seems like it should be possible to get rid of them?

Reply:
@xla might have addressed some of it here: https://github.com/interchainio/tendermint-rs/pull/122

Reply:
https://github.com/interchainio/tendermint-rs/pull/127